stories/Button.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './button.css';
+
+export interface ButtonProps {
+  /**
+   * Is this the principal call to action on the page?
+   */
+  primary?: boolean;
+  /**
+   * What background color to use
+   */
+  backgroundColor?: string;
+  /**
+   * How large should the button be?
+   */
+  size?: 'small' | 'medium' | 'large';
+  /**
+   * Button contents
+   */
+  label: string;
+  /**
+   * Optional click handler
+   */
+  onClick?: () => void;
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Button: React.FC<ButtonProps> = ({
```

Comment:
Why is the button itself in stories?

I think you want the folder structure to be :


```
components/
  button/
     internals/
        css_file
        other_files
     index.tsx
     stories.tsx
```

Reply:
The goal of the story book is to show case app components so that design and dev team can speak the same language